### PR TITLE
Feature/update external dependencies

### PR DESCRIPTION
**Pull request by melsner11Ons**
### What

Updated minor versions of dependencies and plugins
Updated one method call as new library version changed name of method slighly

### How to review

Check out this branch
Add this snippet to your plugin section in your pom.xml file
```
<plugin>
                <groupId>org.codehaus.mojo</groupId>
                <artifactId>versions-maven-plugin</artifactId>
                <version>2.16.2</version>
</plugin>
```
Be sure you have current java 1.8 installed and is being used  by maven. Maven uses the JDK from the current JAVA_HOME environment variable. You can set JAVA_HOME to  your java jdk 1.8 home.

for example on mac os when you installed jdk1.8 with brew : export JAVA_HOME=/usr/local/Cellar/openjdk@8/1.8.0-392/
execute:

mvn versions:display-dependency-updates -DallowMajorUpdates=false
mvn versions:display-plugin-updates

This should not bring up any new updates.
`mvn test` and `mvn package should compile without error`

### Who can review

member of ONS

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -65,29 +65,29 @@
...
-            <version>0.9.12</version>
+            <version>0.10.2</version>
         </dependency>
 
         <!-- Logging -->
         <dependency>
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-api</artifactId>
-            <version>1.7.36</version>
+            <version>1.8.0-beta4</version>
```
Comment (jmgq) [inline pom.xml:90]: I think we need to be careful when deciding to use beta versions. In this case, I would revert the changes, since it looks like the 1.8 branch was abandoned in 2019, whereas the 1.7.36 version is from 2022.

**Review by jmgq — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (melsner11Ons) [reply pom.xml:90]: yes, agree that's a valid point

**Review by lindenmckenzie — APPROVED** [review]

**Review by jmgq — APPROVED** [review]
